### ISSUE TITLE
Massive failure in `cuts_basic`

### ISSUE THREAD

**Issue by FaroutYLq** [XENONnT/reprox#125]
`/project/lgrandi/xenonnt/data_management_reprocessing/job_logs/049655.txt` as an example here:
```
strax.mailbox.MailboxKilled: (<class 'TypeError'>, TypeError("can't unbox array from PyObject into native value.  The object maybe of a different type"), <traceback object at 0x7fef0a689640>)
strax.mailbox.MailboxKilled: (<class 'TypeError'>, TypeError("can't unbox array from PyObject into native value.  The object maybe of a different type"), <traceback object at 0x7fef0a689640>)
strax.mailbox.MailboxKilled: (<class 'TypeError'>, TypeError("can't unbox array from PyObject into native value.  The object maybe of a different type"), <traceback object at 0x7fef0a689640>)
    raise MailboxKilled(self.killed_because)
strax.mailbox.MailboxKilled: (<class 'TypeError'>, TypeError("can't unbox array from PyObject into native value.  The object maybe of a different type"), <traceback object at 0x7fef0a689640>)
    self.close()
strax.mailbox.MailboxKilled: (<class 'TypeError'>, TypeError("can't unbox array from PyObject into native value.  The object maybe of a different type"), <traceback object at 0x7fef0a689640>)
  File "/opt/XENONnT/anaconda/envs/XENONnT_development/lib/python3.9/site-packages/strax/mailbox.py", line 361, in close
    x = next(iterable)
    self.send(StopIteration)
  File "/opt/XENONnT/anaconda/envs/XENONnT_development/lib/python3.9/site-packages/strax/plugins/plugin.py", line 456, in iter
  File "/opt/XENONnT/anaconda/envs/XENONnT_development/lib/python3.9/site-packages/strax/mailbox.py", line 313, in send
    raise MailboxKilled(self.killed_because)
strax.mailbox.MailboxKilled: (<class 'TypeError'>, TypeError("can't unbox array from PyObject into native value.  The object maybe of a different type"), <traceback object at 0x7fef0a689640>)
    self.input_buffer[d].split(
  File "/opt/XENONnT/anaconda/envs/XENONnT_development/lib/python3.9/site-packages/strax/chunk.py", line 170, in split
    data1, data2, t = split_array(
TypeError: can't unbox array from PyObject into native value.  The object maybe of a different type
You specified _auto_append_rucio_local=True and you are not on dali compute nodes,so we will add the following rucio local path:  /project/lgrandi/rucio/
Traceback (most recent call last):
  File "/opt/XENONnT/anaconda/envs/XENONnT_development/bin/straxer", line 352, in <module>
    sys.exit(main(args))
  File "/opt/XENONnT/anaconda/envs/XENONnT_development/bin/straxer", line 299, in main
    run_make(target)
  File "/opt/XENONnT/anaconda/envs/XENONnT_development/bin/straxer", line 268, in run_make
"050111.txt" 400L, 26916C                                                                                                            366,5         94%
  File "/opt/XENONnT/anaconda/envs/XENONnT_development/lib/python3.9/site-packages/strax/mailbox.py", line 281, in _send_from
    raise MailboxKilled(self.killed_because)
    raise MailboxKilled(self.killed_because)
    self.close()
  File "/opt/XENONnT/anaconda/envs/XENONnT_development/lib/python3.9/site-packages/strax/mailbox.py", line 361, in close
    x = next(iterable)
    self.send(StopIteration)
  File "/opt/XENONnT/anaconda/envs/XENONnT_development/lib/python3.9/site-packages/strax/plugins/plugin.py", line 456, in iter
  File "/opt/XENONnT/anaconda/envs/XENONnT_development/lib/python3.9/site-packages/strax/mailbox.py", line 313, in send
    raise MailboxKilled(self.killed_because)
strax.mailbox.MailboxKilled: (<class 'TypeError'>, TypeError("can't unbox array from PyObject into native value.  The object maybe of a different type"), <traceback object at 0x7fef0a689640>)
    self.input_buffer[d].split(
  File "/opt/XENONnT/anaconda/envs/XENONnT_development/lib/python3.9/site-packages/strax/chunk.py", line 170, in split
    data1, data2, t = split_array(
TypeError: can't unbox array from PyObject into native value.  The object maybe of a different type
You specified _auto_append_rucio_local=True and you are not on dali compute nodes,so we will add the following rucio local path:  /project/lgrandi/rucio/
Traceback (most recent call last):
  File "/opt/XENONnT/anaconda/envs/XENONnT_development/bin/straxer", line 352, in <module>
    sys.exit(main(args))
  File "/opt/XENONnT/anaconda/envs/XENONnT_development/bin/straxer", line 299, in main
    run_make(target)
  File "/opt/XENONnT/anaconda/envs/XENONnT_development/bin/straxer", line 268, in run_make
    for i, d in enumerate(get_results()):
  File "/opt/XENONnT/anaconda/envs/XENONnT_development/bin/straxer", line 265, in get_results
    yield from st.get_iter(**kwargs)
  File "/opt/XENONnT/anaconda/envs/XENONnT_development/lib/python3.9/site-packages/strax/context.py", line 1324, in get_iter
    generator.throw(e)
  File "/opt/XENONnT/anaconda/envs/XENONnT_development/lib/python3.9/site-packages/strax/context.py", line 1296, in get_iter
    for n_chunks, result in enumerate(strax.continuity_check(generator), 1):
  File "/opt/XENONnT/anaconda/envs/XENONnT_development/lib/python3.9/site-packages/strax/chunk.py", line 303, in continuity_check
    for chunk in chunk_iter:
  File "/opt/XENONnT/anaconda/envs/XENONnT_development/lib/python3.9/site-packages/strax/processor.py", line 302, in iter
    raise exc.with_traceback(traceback)
  File "/opt/XENONnT/anaconda/envs/XENONnT_development/lib/python3.9/site-packages/strax/mailbox.py", line 281, in _send_from
    x = next(iterable)
  File "/opt/XENONnT/anaconda/envs/XENONnT_development/lib/python3.9/site-packages/strax/plugins/plugin.py", line 456, in iter
    self.input_buffer[d].split(
  File "/opt/XENONnT/anaconda/envs/XENONnT_development/lib/python3.9/site-packages/strax/chunk.py", line 170, in split
    data1, data2, t = split_array(
TypeError: can't unbox array from PyObject into native value.  The object maybe of a different type
```

**Comment by FaroutYLq** [comment]
This only becomes a problem in `reprox` or `straxer`. The same run's `cuts_basic` can be processed in a notebook with no problem. 
```
st.make('049655', 'cuts_basic')
```

**Comment by FaroutYLq** [comment]
Environment for `reprox`:
```
 module version                                                                               path  git
 python  3.9.16                          /opt/XENONnT/anaconda/envs/XENONnT_development/bin/python None
  strax   1.5.1   /opt/XENONnT/anaconda/envs/XENONnT_development/lib/python3.9/site-packages/strax None
straxen   2.1.0 /opt/XENONnT/anaconda/envs/XENONnT_development/lib/python3.9/site-packages/straxen None
  cutax  1.15.1                              /home/yuanlq/.local/lib/python3.9/site-packages/cutax None
```
On notebook: 
<img width="591" alt="image" src="https://github.com/XENONnT/reprox/assets/47046530/978756b0-45e2-4a20-ba4e-eabea20c2842">


module | version | path | git
-- | -- | -- | --
python | 3.9.16 | /opt/XENONnT/anaconda/envs/XENONnT_development... | None
strax | 1.5.1 | /cvmfs/xenon.opensciencegrid.org/releases/nT/d... | None
straxen | 2.1.0 | /cvmfs/xenon.opensciencegrid.org/releases/nT/d... | None
cutax | 1.15.1 | /home/yuanlq/.local/lib/python3.9/site-package... | None




module | version | path | git
-- | -- | -- | --
python | 3.9.16 | /opt/XENONnT/anaconda/envs/XENONnT_development... | None
strax | 1.5.1 | /cvmfs/xenon.opensciencegrid.org/releases/nT/d... | None
straxen | 2.1.0 | /cvmfs/xenon.opensciencegrid.org/releases/nT/d... | None
cutax | 1.15.1 | /home/yuanlq/.local/lib/python3.9/site-package... | None

**Comment by FaroutYLq** [comment]
Tried to downgrade cutax to commit 34731dbb96db90b6c941a83b15a441a922872b7f. Got same error.

**Comment by FaroutYLq** [comment]
Draft solution below failed... no idea why 
<img width="816" alt="image" src="https://github.com/XENONnT/reprox/assets/47046530/c7de6175-132e-414e-8316-0390d742d26b">

**Comment by FaroutYLq** [comment]
Checked `numpy` and `numba` version. No clue
```
# straxer
>>> print(np.__file__)
/cvmfs/xenon.opensciencegrid.org/releases/nT/development/anaconda/envs/XENONnT_development/lib/python3.9/site-packages/numpy/__init__.py
>>> print(np.__version__)
1.22.4
>>> print(numba.__file__)
/cvmfs/xenon.opensciencegrid.org/releases/nT/development/anaconda/envs/XENONnT_development/lib/python3.9/site-packages/numba/__init__.py
>>> print(numba.__version__)
0.55.2

# notebook
/cvmfs/xenon.opensciencegrid.org/releases/nT/development/anaconda/envs/XENONnT_development/lib/python3.9/site-packages/numpy/__init__.py
1.22.4
/cvmfs/xenon.opensciencegrid.org/releases/nT/development/anaconda/envs/XENONnT_development/lib/python3.9/site-packages/numba/__init__.py
0.55.2
```

**Comment by FaroutYLq** [comment]
Seems temporarily fixed by #128 `--notlazy`. The mechanism is unclear but let's close it for now...
